packages/react-native-branch/android/src/main/java/expo/modules/adapters/branch/BranchReactActivityLifecycleListener.kt

```diff
@@ -2,12 +2,21 @@ package expo.modules.adapters.branch
 
 import android.app.Activity
 import android.content.Context
+import android.content.Intent
 import android.os.Bundle
 import expo.modules.core.interfaces.ReactActivityLifecycleListener
-import io.branch.rnbranch.*
+import io.branch.rnbranch.RNBranchModule
+
 
 class BranchReactActivityLifecycleListener(activityContext: Context) : ReactActivityLifecycleListener {
     override fun onCreate(activity: Activity, savedInstanceState: Bundle?) {
         RNBranchModule.initSession(activity.getIntent().getData(), activity);
     }
+
+    override fun onNewIntent(intent: Intent?): Boolean {
+        if (intent != null) {
+            RNBranchModule.onNewIntent(intent);
+        }
+        return false
```

Comment:
Don't know which value should return, `false` or `true`?

Reply:
maybe `false` here and let [`ReactActivityDelegate.onNewIntent`](https://github.com/facebook/react-native/blob/c68c47d2bafa8e8e25b534d6cdd1a63bc77a1cf4/ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java#L144-L150) to handle the return value.